flask_restalchemy/serialization/fields.py

```diff
@@ -131,6 +163,13 @@ def __init__(self, declarative_class, **kw):
         super().__init__(serializer=self.PrimaryKeySerializer(declarative_class), **kw)
 
 
+def get_pk_attr_name(declarative_model):
```

Comment:
docs: _Get the primary key attribute name from a Declarative model class_

Reply:
done